src/main/java/frc/robot/subsystems/Arm.java

```diff
@@ -359,7 +357,7 @@ private void checkPivotCalibration() {
     }
 
     public boolean isCalibrated() {
-        return ((true == isPivotCalibrated) && (true == isExtenderCalibrated));
+        return ((isPivotCalibrated) && (isExtenderCalibrated));
```

Comment:
Can we revert this conditional so it looks similar to the previous ones. Helps newer coders get used to the boolean == variable conditionals.

Reply:
Why are they true ==, not e == true

Reply:
with variable == true, people can accidentally make the conditional set the variable if they only use 1 equal sign.
Ex: variable = true.
If you do true = variable, it makes the error more obvious because true isn't a variable and thus can't be set.
Therefore we use true == variable so people don't accidentally use 1 equals sign and not realize it.

Reply:
Done